dist/profile/manifests/mirrorbrain.pp

```diff
@@ -28,6 +28,7 @@
   }
 
   include profile::apachemisc
+  include profile::azure
```

Comment:
Does it make sense to include profile::azure in mirrorbrain if it's already defined in base?